rspamd.service

```diff
@@ -9,9 +9,39 @@ NonBlocking=true
 ExecStart=/usr/bin/rspamd -c /etc/rspamd/rspamd.conf -f
 ExecReload=/bin/kill -HUP $MAINPID
 User=_rspamd
+ConfigurationDirectory=rspamd
 RuntimeDirectory=rspamd
 RuntimeDirectoryMode=0755
+StateDirectory=rspamd
+StateDirectoryMode=0700
+LogsDirectory=rspamd
 Restart=always
 
+AmbientCapabilities=
+CapabilityBoundingSet=
```

Comment:
Start the service without any capabilities such as `CAP_SYS_ADMIN`, `CAP_DAC_OVERRIDE`...